ro.go.hmlendea.Duolingo.yaml

```diff
@@ -0,0 +1,51 @@
+app-id: ro.go.hmlendea.Duolingo
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+separate-locales: false
+command: start-duolingo-desktop
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+
+modules:
+  - name: duolingo-desktop
+    buildsystem: simple
+    build-commands:
+      - unzip -d duolingo-desktop duolingo-desktop_*.zip
+      - rm duolingo-desktop_*.zip
+      - cp -r duolingo-desktop /app
+      - install -Dm 644 ro.go.hmlendea.Duolingo.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -Dm 644 ro.go.hmlendea.Duolingo.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm 644 ro.go.hmlendea.Duolingo.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.appdata.xml
+      - install -Dm 755 start-duolingo-desktop.sh /app/bin/start-duolingo-desktop
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://github.com/hmlendea/duolingo-desktop/releases/download/v2.0.10/duolingo-desktop_2.0.10_linux.zip
+        sha256: b9f9b09b1f79a847252dc0fdec323dffe2f8193059da81e2eae2a344c4e7f326
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/hmlendea/duolingo-desktop/releases/latest
+          version-query: .tag_name | sub("^v"; "")
+          url-query: .assets[] | select(.name=="duolingo-desktop_" + $version +
+            "_linux.zip") | .browser_download_url
+      - type: file
+        path: ro.go.hmlendea.Duolingo.metainfo.xml
+      - type: file
+        path: ro.go.hmlendea.Duolingo.png
```

Comment:
the icon being in the upstream directory you should add here. If it's not in the tarball, you should get it from the repository.

Reply:
this file is no longer used and should removed or used